src/impl/locale.js

```diff
@@ -123,10 +148,9 @@ function intlConfigString(localeStr, numberingSystem, outputCalendar) {
     if (numberingSystem) {
       localeStr += `-nu-${numberingSystem}`;
     }
-    return localeStr;
-  } else {
-    return localeStr;
   }
+  intlConfigCache[key] = localeStr;
```

Comment:
I'm having a hard time understanding how this would give such a big speedup. Can you use the benchmarks in the repo to demonstrate this?